whereami/k8s-frontend-overlay-example/cm-flag.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: whereami-configmap
+data:
+  BACKEND_ENABLED: "True" 
+  BACKEND_SERVICE: "whereami-backend" # substitute with corresponding service name - this example assumes both services are in the same namespace  
+  METADATA:        "frontend"
```

Comment:
@mark-church I'm seeing some extra whitespace in line 8

Reply:
fixed the whitespace